dbcon/mysql/ha_mcs_impl.cpp

```diff
@@ -4326,15 +4312,12 @@ int ha_mcs_impl_group_by_next(TABLE* table)
 {
     THD* thd = current_thd;
 
-    if (((thd->lex)->sql_command == SQLCOM_UPDATE)  || ((thd->lex)->sql_command == SQLCOM_DELETE) ||
-            ((thd->lex)->sql_command == SQLCOM_DELETE_MULTI) || ((thd->lex)->sql_command == SQLCOM_UPDATE_MULTI))
+    if ((thd->lex->sql_command == SQLCOM_UPDATE) ||
```

Comment:
Is it possible to replace this repeating conditions with the bool func you created?

Reply:
Not possible in this case since the following `&&` goes together:
`(thd->lex->sql_command == SQLCOM_UPDATE_MULTI && !isForeignTableUpdate(thd))`